.github/workflows/check-patches.yml

```diff
@@ -1,8 +1,17 @@
+---
```

Comment:
Not a full-time YAML wizard, is this added for a reason or is it by accident?

Reply:
This is the usual YAML header.